beacon_node/src/cli.rs

```diff
@@ -776,4 +776,12 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true)
                 .default_value("true")
         )
+        .arg(
+            Arg::with_name("reset-payload-statuses")
+                .long("reset-payload-statuses")
+                .help("When present, Lighthouse will forget the payload statuses of any \
+                       already-imported blocks. This can assist in the recovery from a consensus \
+                       failure caused by the execution layer.")
+                .takes_value(true)
```

Comment:
```suggestion
                .takes_value(false)
```

Just noticed this while lurking